Source/Krypton Components/Krypton.Toolkit/View Draw/ViewDrawMenuItem.cs

```diff
@@ -405,26 +402,23 @@ public void ShowSubMenu(bool keyboardActivated)
                     _contextMenu.Disposed += OnContextMenuDisposed;
 
                     // Get the screen rectangle for the drawing element
-                    if (OwningControl != null)
+                    Rectangle menuDrawRect = OwningControl.RectangleToScreen(ClientRectangle);
```

Comment:
Did you mean to back out these changes as well ?

Reply:
It was easier to roll back and reintroduce the fixes